pkg/capi/edit/cluster.x-k8s.io.cluster/ClusterConfig.vue

```diff
@@ -362,44 +408,56 @@ export default {
           />
         </div>
       </div>
-      <div class="row row-config">
-        <div class="col span-12 mt-20">
+      <div class="row span-12 row-config">
+        <div class="col span-6 mt-20">
           <h2>
             <t k="capi.cluster.controlPlaneEndpoint.title" />
           </h2>
           <ControlPlaneEndpointSection
             v-model:value="controlPlaneEndpoint"
             :mode="mode"
             :rules="{host:fvGetAndReportPathRules('spec.controlPlaneEndpoint.host'), port:fvGetAndReportPathRules('spec.controlPlaneEndpoint.port') }"
-            @update:value="$emit('update:value', {k: 'spec.controlPlaneEndpoint', val: $event})"
+            @control-plane-endpoint-changed="controlPlaneEndpointChanged"
+          />
+        </div>
+        <div class="col span-6 mt-20">
+          <h2>
+            <t k="capi.cluster.topology.controlPlane.title" />
+          </h2>
+          <ControlPlaneSection
+            v-model:value="topology"
```

Comment:
This component only cares about replicas; would it be possible to pass in only the replica field? It would be easier to tell, when reading ClusterConfig, what ControlPlaneSection actually does if it had a `replicas` property and emitted an `update:replicas` event